test/framework/build.gradle

```diff
@@ -93,5 +93,5 @@ tasks.register("verifyVersions") {
 
 tasks.named('splitPackagesAudit').configure {
   // for now we always run tests with the classpath, so we are ok with split packages for tests
-  onlyIf { false }
+  onlyIf("always false") { false }
```

Comment:
Yeah, these are awkward. I'm wondering if `enabled = false` actually might be better than showing a skip message of "'always false' not satisfied". That's pretty confusing.

Reply:
This gives us `'Task is enabled' not satisfied` which seems okay I think

Reply:
Yeah, I think that's better than `'always false' not satisfied`.